index.js

```diff
@@ -222,6 +223,29 @@ var putFile = function (bucket, key, filepath) {
     });
 };
 
+/* Put an object in S3
+ *
+ * Parameters:
+ *  any valid parameters listed that aws-sdk.s3().upload() supports
+ *  https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#upload-property
```

Comment:
The only downside to this is that all consumers have to know the parameter listing of the api. If S3 changed their api, we would have to update all consumers or put a translator in here. Not a big deal, purely preference. We could address this when we replace this package.

Reply:
Ya I hear ya.  I was treating this change as a one off case since we will hopefully do away with this whole package sooner rather than later